### ISSUE TITLE
Order of usage example arguments does not match argument 'help' list ordering

### ISSUE THREAD

**Issue by bevanweiss** [commandlineparser/commandline#646]
This is one of those completely minor items, but I think a slight usability improvement.

When adding usage examples, the ordering of the arguments appear to be in alphabetic order as per argument name.
This doesn't match the ordering of the arguments in the 'help list' where they are listed by defined order (which I personally refer).

It would be great if the usage example ordering matched the 'help list' ordering.

**Comment by Jusas** [comment]
I agree, this would be a nice improvement. Let's say I have primary arguments that define the main operation, and then minor detail arguments that are mere tweaks to the program operation (such as --verbose etc), I do want my main defining arguments to be listed first in the examples. This is specifically noticeable with parameter sets, like for example `--use-manual-dimensions --width 50 --height 40` where it may look funny if the order is for example `--height 40 --width 50 --use-manual-dimensions` as the width and height arguments actually depend on the --use-manual-dimensions being set.
